bertopic/_bertopic.py

```diff
@@ -2339,6 +2339,95 @@ def visualize_documents(self,
                                             width=width,
                                             height=height)
 
+    def visualize_document_datamap(self,
+                                   docs: List[str],
+                                   topics: List[int] = None,
+                                   embeddings: np.ndarray = None,
+                                   reduced_embeddings: np.ndarray = None,
+                                   custom_labels: Union[bool, str] = False,
+                                   title: str = "Documents and Topics",
+                                   sub_title: Union[str, None] = None,
+                                   width: int = 1200,
+                                   height: int = 1200,
+                                   **datamap_kwds):
+        """ Visualize documents and their topics in 2D as a static plot for publication using
+        DataMapPlot. This works best if there are between 5 and 60 topics. It is therefore best
+        to use a sufficiently large `min_topic_size` or set `nr_topics` when building the model.
```

Comment:
What do you think about adding a `topics` parameter that allows you to select a subset of topics to visualize? That way, users are not limited by needing to generate between 5 and 60 topics but can freely select whichever topics they want visualized. All other topics would then be shown as -1/outliers/unlabelled I guess.

Reply:
That's a reasonable option, and I agree that it makes sense. Let me see if I can make that work.